effectiveOfficeBackend/src/main/kotlin/office/effective/plugins/Migration.kt

```diff
@@ -0,0 +1,36 @@
+package office.effective.plugins
+
+import io.ktor.server.application.*
+import liquibase.Liquibase
+import liquibase.database.DatabaseFactory
+import liquibase.database.jvm.JdbcConnection
+import liquibase.resource.ClassLoaderResourceAccessor
+import office.effective.config
+import java.sql.DriverManager
+
+fun Application.configureMigration() {
+    val url: String = config.propertyOrNull("database.url")
+        ?.getString() ?: "jdbc:postgresql://localhost:15432/effectiveOfficeBackendDB"
+    val username: String = config.propertyOrNull("database.username")?.getString() ?: "postgres"
+    val password: String = config.propertyOrNull("database.password")?.getString() ?: "test1234567890"
+
+    val changelogFile: String = config.propertyOrNull("liquibase.changelogFile")
+        ?.getString() ?: "changelog/changelog-master.yaml"
+    val defaultSchemaName: String = config.propertyOrNull("liquibase.defaultSchemaName")
+        ?.getString() ?: "public"
```

Comment:
It's worth making it so that if a variable is null, its value will say so, not just "test12345", perhaps such a string value should also be put into some kind of constant.